# Motion of Astronomical Bodies

## Orbital Periods

Copernicus determined the scale of the Solar System from observations, in particular using the *synodic period* $S$ and sidereal period $P$.  He didn't actually know the distance from the Earth to the Sun, so he (and others) let the Earth-Sun distance equal to $1$ (unity).
    - [**Astronomical unit**](https://en.wikipedia.org/wiki/Astronomical_unit) (AU): the distance from the Earth to the Sun
    - During Copernicus' time, the AU represented the actual distance; Later, it was redefined to be the *average* distance.  Today it's defined as fundamental constants (i.e., the speed of light).

The sidereal period $P$ of a planet can be determined by comparing the planet's synodic period $S$ to the Earth's sidereal period $E = 365.25\ {\rm days} = 1\ {\rm yr}$.  However this comparison differs depending on whether the planet (in question) is on a superior or inferior orbit compared to the Earth.  The relations are given as

\begin{align*}
\frac{1}{P} &= 
    \begin{cases} 
        \frac{1}{E} + \frac{1}{S}, \quad \text{(inferior)} \\
        \frac{1}{E} - \frac{1}{S}. \quad \text{(superior)}
    \end{cases}
\end{align*}

For Saturn (a superior orbit planet), the time that passes between oppositions (or the time between maximum brightness) is $378.1\ {\rm days} = 1.035\ {\rm yr}$.  Then we can calculate Saturn's sidereal period as

\begin{align*}
\frac{1}{P} &= \frac{1}{1\ {\rm yr}} - \frac{1}{1.035\ {\rm yr}} = 0.03399\ {\rm yr^{-1}}, \\
P &= \frac{1}{0.03399\ {\rm yr^{-1}}} = 29.42\ {\rm yr}.
\end{align*}

In [1]:
E = 365.25 #Earth's sidereal period (in days)
S = 378.1 #Saturn's synodic period (in days)

P_inv = 1- (E/S) #1/P (Saturn's siderial period)
P = 1/P_inv 

print("Saturn's sidereal period is %2.1f yr." % P)

Saturn's sidereal period is 29.4 yr.



**Saturn's sidereal period is 29.4 years**, which means that Saturn takes 29.4 years to return to the same spot while orbiting the Sun.

In Copernicus' model, the planetary orbits were perfect circles.  

- He needed to measure the time $t_O$ when a planet was at opposition (for outer planets) or inferior conjunction (for inner planets).
- The time $t$ when the same planet was at quadrature (for outer planets) or elongation (for inner planets).  

The difference in these times ($T = |t-t_O|$) would tell Copernicus what fraction of an orbit the planet had moved.  This would provide an estimate for the heliocentric longitude $\lambda$ of the planet at quadrature or elongation.  See [here](https://astro.unl.edu/naap/ssm/ssm_advanced.html) for more details.

$$ \lambda = \left(\frac{T}{S}\right) \times 360^\circ. $$

Recall that the special points (quadrature or elongation) form right triangles, where one angle is $90^\circ$ and another angle will be the heliocentric longitude $\lambda$.  In both cases, one side of the triangle will be the Earth-Sun distance (i.e., 1 AU).  Copernicus only need to use trigonometry to find the side corresponding to the planet-Sun distance $d_p$.  The following relations describe the trigonometry required for a planet on an inferior or superior orbit:

\begin{align*}
d_p &= \begin{cases}
    \cos{\lambda}, \quad \text{(inferior)} \\
    \frac{1}{\cos{\lambda}}. \quad \text{(superior)}
    \end{cases}
\end{align*}

For Saturn, it takes ~88 days (0.241 yr) for Saturn to move from opposition to next quadrature because Earth is moving along its orbit too.  Using this estimate, we have 
$$\lambda_{\rm Sat} = \left(\frac{0.241\ {\rm yr}}{1.035\ {\rm yr}}\right) \times 360^\circ = 83.8^\circ.$$

The Saturn-Sun distance is then 

$$d_{\rm Sat} = \frac{1\ {\rm AU}}{\cos{(83.8^\circ)}} = 9.3\ {\rm AU}.$$

**Not too far from the modern value, $9.58\ {\rm AU}$.**

In [2]:
import numpy as np

S = 1.035 #Saturn's synodic period (in yr)

T = 0.241 # Time between Saturn's opposition and next quadrature (in yr)
lamb = (T/S)*360. #heliocentric longitude (in degrees)

print("Saturn's heliocentric longitude at quadrature is %2.1f deg." % lamb)

d_sat = 1/np.cos(np.radians(lamb))
print("The Saturn-Sun distance is %1.1f AU." % d_sat)


Saturn's heliocentric longitude at quadrature is 83.8 deg.
The Saturn-Sun distance is 9.3 AU.


## Kepler's 3rd Law

## Using Newton's Laws